upgrade/helpers/tasks.py

```diff
@@ -1284,3 +1272,19 @@ def job_execution_time(task_name, start_time=None):
     else:
         start_time = datetime.now().replace(microsecond=0)
         return start_time
+
+
+def resume_failed_task():
+    """
+    This function is used to resume the canceled paused tasks.
+    """
+    output = run("hammer task list|awk '/Task canceled/ && /paused/ {print $1}'")
+    run(f"hammer task resume --task-ids {','.join(output.split())}")
+    logger.info(f"The CV publish task {','.join(output.split())} has resumed successfully, "
+                f"waiting for their completion")
+    for task_id in output.split():
+        run(f'hammer task progress --id {task_id}')
```

Comment:
Why do you want to see the progress ? Is it for monitoring the job ? Also, is this command is `async` which wont hold ur program until the progress bar completes for each task?

Reply:
Some of the further tasks depend on their completion(on dependent cv task), so it is required, This command is async.